pkgs/applications/misc/streamdeck-ui/update-pillow.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/pyproject.toml b/pyproject.toml
-index 0aff29e..4371616 100644
+index 54a8c19..db1f64c 100644
 --- a/pyproject.toml
 +++ b/pyproject.toml
 @@ -14,7 +14,7 @@ packages = [
  [tool.poetry.dependencies]
- python = ">=3.8,<3.12"
- streamdeck = "^0.9.3"
--pillow = "^9.4.0"
-+pillow = "^10.0.0"
- pynput = "^1.7.6"
+ python = ">=3.8,<3.13"
+ streamdeck = "^0.9.5"
+-pillow = "10.2.0"
++pillow = "^10.2.0"
```

Comment:
This patch is going to conflict continuously. Consider:
https://nixos.org/manual/nixpkgs/stable/#using-pythonrelaxdepshook

Reply:
Did not know about this, nice